### DISCUSSION TITLE
fa/manager/client-software-on-ios/Tutorial-for-Streisand/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4391] [Content Help کمک به اصلاح محتوا]
# fa/manager/client-software-on-ios/Tutorial-for-Streisand/

The proxy manager platform.

https://hiddify.com/fa/manager/client-software-on-ios/Tutorial-for-Streisand/

**Comment by IRTornado** [comment]
سلام بنده یه مشکلی دارم و اونم اینکه با این برنامه روی ای او اس با هر کانفیگ و اینترنت نمیتونم فایل روی تلگرام اپلود کنم همه برنامه اپدیت و تلگرام پاک شده و دوباره نصب شدهمه تنظیمات انجام شده ولی به هیچ عنوان فایل نمیتونم برا یکسی بفرستم ممنون میشم راهنمایی کنید
